integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -151,50 +160,36 @@ public void testCreateSecondOperatorUsingSameOperatorNsNegativeInstall() throws
     Operator secondoperator = new Operator(operatorMap, false, true, true, RestCertType.SELF_SIGNED);
     String oprelease = (String)(secondoperator.getOperatorMap()).get("releaseName");
     String opnamespace = (String)(secondoperator.getOperatorMap()).get("namespace");
+    Exception installException = null;
+    Exception infoException = null;
     try {
       secondoperator.callHelmInstall();
-      throw new RuntimeException(
-          "FAILURE: Helm installs second operator with same namespace as the first one");
-
+      installException = new Exception(
+          "FAILURE: Second Operator Helm Installation MUST fail");
     } catch (Exception ex) {
       if (!ex.getMessage()
-          .contains(
-              "Error: release "
-                  + oprelease
-                  + " failed: secrets \"weblogic-operator-secrets\" already exists")) {
-        throw new RuntimeException(
-            "FAILURE: Helm installs second operator with same namespace as the first one "
-                + "does not report expected message "
+          .contains("weblogic-operator-secrets")) {
+        infoException = new Exception(
+            "Exception does not report expected message "
                 + ex.getMessage());
       }
-      String cmdLb = "";
-      if (BaseTest.HELM_VERSION.equals("V2")) {
-        cmdLb = "helm list --failed " + "  | grep " + oprelease;
-      }
-      if (BaseTest.HELM_VERSION.equals("V3")) {
-        cmdLb = "helm list --failed --namespace " + opnamespace + "  | grep " + oprelease;
-      }
-      LoggerHelper.getLocal().log(Level.INFO, "Executing cmd " + cmdLb);
-      ExecResult result = ExecCommand.exec(cmdLb);
-      if (result.exitValue() != 0) {
-        throw new RuntimeException(
-            "FAILURE: Helm installs second operator with same namespace as the first one ");
-      }
+      LoggerHelper.getLocal().log(Level.INFO, "Exception Messsage " + ex.getMessage());
     } finally {
       if (firstoperator != null) {
         firstoperator.destroy();
       }
-      if (secondoperator != null) {
-        secondoperator.destroy();
-      }
     }
+    Assertions.assertNull(installException,"Second Operator Helm Installation MUST fail");
```

Comment:
space after comma
static import Assertions

Reply:
operator should not have a capital

Reply:
done